package.json

```diff
@@ -94,8 +97,9 @@
   "scripts": {
     "postinstall": "./tools/task-runner/runner githooks --verbose",
     "sass-watch": "node-sass -w ./static/src/stylesheets -o ./static/target/stylesheets --source-map=true",
-    "css-watch": "gulp --cwd ./dev watch:css",
+    "babel-watch": "BABEL_ENV=production babel ./static/src/javascripts --watch -d ./static/transpiled/javascripts --ignore .eslintrc.js >/dev/null 2>&1",
```

Comment:
hmmmm, actually watch should probably be dev, and dev updated to match prod. things like react change how they're built (debug info etc), and that would be useful in dev